linux/centos/7/foxpass_setup.py

```diff
@@ -26,6 +26,7 @@
 
```

Comment:
Thanks for doing this Bryan. This looks good. Diff gives a good idea to the user regarding the changes made by the script. In fact we could also have a `--dry-run` which provides the changes that will be done when executed in real. But this is good for now :)